dotCMS/src/main/java/com/dotcms/rest/api/v1/system/ConfigurationHelper.java

```diff
@@ -36,6 +36,7 @@
 public class ConfigurationHelper implements Serializable {
 
 	public static final String EDIT_CONTENT_STRUCTURES_PER_COLUMN = "EDIT_CONTENT_STRUCTURES_PER_COLUMN";
+	public static final String DOTSITESELECTOR_SITES_PER_PAGE = "DOTSITESELECTOR_SITES_PER_PAGE";
```

Comment:
That property should not be specific to the site selector, we could use that property on more selects.